SingularityUI/app/views/tail.coffee

```diff
@@ -142,16 +148,20 @@ class TailView extends View
         @startTailing()
 
     startTailing: =>
-        return if @isTailing is true
+        return if @isTailing or not @isTaskRunning
 
         @isTailing = true
         @scrollToBottom()
 
         clearInterval @tailInterval
         @tailInterval = setInterval =>
+            @stopTailing() if not @isTaskRunning
+
             @collection.fetchNext().done =>
                 # Only show the newly tail-ed lines if we are still tailing
                 @scrollToBottom() if @isTailing
+                @model.fetch()
```

Comment:
We can do that, if we use the global refresh, we just still need to check at `startTailing` as that can still be run after the model is changed (after `initialdata` event and a couple other actions).